### DISCUSSION TITLE
Top Solutions do not look like the target

### DISCUSSION THREAD

**Discussion by axelf** [cssbattle/forum#362] [General]
Hi,

the top solutions from yesterday daily target 22/10/2023 don't look like the target image. I think this is not the idea to submit something completely different. This is only 1 example.
![cssbattle](https://github.com/cssbattle/forum/assets/1690324/f322c072-b683-4473-96ed-987b12494d6a)

**Comment by PavButsa** [comment]
Hi! Top solution looks 100% identical in last version of Chrome browser.
You can also use [this previewer](https://tc70f3.csb.app) to check solutions.

**Reply by ngdangtu-vn** [reply]
They should add this link next to every top solution

**Reply by axelf** [reply]
Then you should also write on the website that we must use only the latest version of chrome. I think it's not the goal to learn css for only one browser

**Reply by ngdangtu-vn** [reply]
Not learning, but understanding. Is there something wrong with it?

**Reply by axelf** [reply]
Nothing wrong with it but in a real world project you can't force the user to use only one special version of a browser.

**Reply by ngdangtu-vn** [reply]
No one say you can or should, are you saying you want to apply these things to real world application?

**Reply by axelf** [reply]
Ok, lets close this discussion. That's a waste of time. You completely don't understand my point.

**Comment by Tryskall** [comment]
I have to say that writing code for only one browser is not a great way to code. These solutions should work in as many browsers as possible. So, when checking the solution, it should be checked among at least three browsers. To allow solutions that only work on one browser just doesnt make sense. I will add this post to suggestions for new updates.
